systems/overview/12-Glossary.md

```diff
@@ -51,3 +51,7 @@ Wikitext, also known as Wiki markup or Wikicode, consists of the syntax and keyw
 ## [MediaWiki Core](https://www.mediawiki.org/wiki/Core)
 
 MediaWiki Core refers to all files that are part of the main MediaWiki package.
+
+## [Special Page](https://www.mediawiki.org/wiki/Manual:Special_pages)
```

Comment:
Should we just link to or at least used the same wording as on https://www.mediawiki.org/wiki/Special_pages ?

Reply:
I don't think a simple link would fit the purpose of a Glossary. I slightly modified the wording from https://www.mediawiki.org/wiki/Special_pages in an attempt to shorten it, but am happy to use exactly that, when you think we should stick to the original.